src/java.base/share/classes/sun/net/www/MimeLauncher.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1994, 1998, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1994, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Is MimeEntry.launch still used? I'm wondering if the MimeLauncher can be deleted.